src/map/utils/battleutils.cpp

```diff
@@ -2235,6 +2235,12 @@ namespace battleutils
         // Apply TP 
         PChar->addTP(std::max((PChar->getMod(Mod::SAVETP)), standbyTp));
 
+        // Remove Hagakure Effect if present
+	if (PChar->StatusEffectContainer->HasStatusEffect(EFFECT_HAGAKURE))
```

Comment:
indent is off. 4 spaces per indent, brace on newline aligned withs tart of conditionals